jobs/JGDAS_GLOBAL_OCEAN_ANALYSIS_VRFY

```diff
@@ -0,0 +1,45 @@
+#!/bin/bash
+export STRICT="NO"
+source "${HOMEgfs}/ush/preamble.sh"
+
+
+export DATA="${DATAROOT}/${RUN}ocnanal_${cyc}"
+source "${HOMEgfs}/ush/jjob_header.sh" -e "ocnanalrun" -c "base ocnanal ocnanalprep"
```

Comment:
Is this job going to use the same resources of `ocnanalrun`? I highly doubt that.

Reply:
oops, nope.